ViAn/GUI/mainwindow.cpp

```diff
@@ -635,12 +635,15 @@ void MainWindow::on_actionAddVideo_triggered() {
         project = ui->project_tree->selectedItems().first();
         MyQTreeWidgetItem *my_project = (MyQTreeWidgetItem*) project;
         if (my_project->type == TYPE::PROJECT){
-            QString dir = QFileDialog::getOpenFileName(this, tr("Choose video"), this->fileHandler->get_work_space().absolutePath().toStdString().c_str(),
+            Project *proj = this->fileHandler->get_project(my_project->id);
+            std::string video_map_path = this->fileHandler->get_dir(proj->dir_videos).absolutePath().toStdString();
```

Comment:
`video_map_path` seems an odd variable name. Maybe `video_dir_path` would be better.